src/components/UI/Text/Textarea.vue

```diff
@@ -206,15 +208,14 @@ export default {
   methods: {
     //update HTML on input change in textarea
```

Comment:
same comments as in `InputText.vue` component.
- write a proper docstring, using syntax `/** ... */`
- write what this method is exactly doing